code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -281,7 +281,7 @@
 			if(91.01 to INFINITY)
 				msg += "[t_He] [t_is] a shitfaced, slobbering wreck.\n"
 
-	if(src != user && user.has_trait(TRAIT_EMPATH) && !appears_dead)
+	if(src != user && user.has_trait(TRAIT_EMPATH) && !appears_dead && !stat == DEAD)

```

Comment:
...the "is the target dead" check is already included in the `appears_dead`, dude, no need to double dip.

Reply:
I'm confused what you mean by add a check for living then. isliving(user) is done at the start of the proc.

Reply:
Oh oops. I got it.